core/src/main/java/org/wso2/testgrid/core/command/GenerateTestPlanCommand.java

```diff
@@ -180,8 +180,11 @@ private void processTestgridConfiguration(String jobConfigFilePath)
     }
 
     private void processTestgridYaml(TestgridYaml testgridYaml, JobConfigFile jobConfigFile)
-            throws CommandExecutionException, IOException, TestGridDAOException {
-        // TODO: validate the testgridYaml. It must contain at least one infra provisioner, and one scenario.
+            throws CommandExecutionException, TestGridDAOException {
+        if (!validateTestgridYaml(testgridYaml)) {
+            throw new CommandExecutionException(
+                    "Invalid tesgridYaml file is found,Please verify the content of the testgridYaml file");
```

Comment:
Shall we break this comment into two lines instead of separating using a ',' ?

Reply:
Fixed 9be59e94046fe2feb8fe36c3e1d77913ba1669a7